nlubridge/vendors/rasa3.py

```diff
@@ -109,7 +113,7 @@ def test(self, dataset: NLUdataset) -> NLUdataset:
             additional attribute 'probs' (List[float]).
         """
         if self.agent is None:
-            logger.error("Rasa3 classifier has to be trained first!")
+            raise Exception("Rasa3 classifier has to be trained first!")
```

Comment:
IN Watson Vendor raisen wir an der Stelle RuntimeError.

Reply:
Guter Punkt, sollten wir konsistent halten!

Reply:
Das würde ich jetzt einfach so mergen und danach ändern.